src/main/java/ua/javaPro/berezhnoy/hillelHomeWork6/phonebook/Phonebook.java

```diff
@@ -0,0 +1,40 @@
+package ua.javaPro.berezhnoy.hillelHomeWork6.phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Phonebook {
+    private final List<Recording> records = new ArrayList<>();
+
+    public Recording recordAdd(String name, String phone) {
```

Comment:
```suggestion
    private Recording createRecording(String name, String phone) {
```
приватний щоб зайвий раз ніхто не використав
створити запис краще відповідає що метод робить.